Userland/Libraries/LibC/langinfo.cpp

```diff
@@ -25,81 +26,47 @@ static const char* __nl_langinfo(nl_item item)
     case PM_STR:
         return "PM";
     case DAY_1:
-        return "Sunday";
     case DAY_2:
-        return "Monday";
     case DAY_3:
-        return "Tuesday";
     case DAY_4:
-        return "Wednesday";
     case DAY_5:
-        return "Thursday";
     case DAY_6:
-        return "Friday";
     case DAY_7:
-        return "Saturday";
+        return reinterpret_cast<char const*>(long_day_names[item - DAY_1].bytes().data());
     case ABDAY_1:
-        return "Sun";
     case ABDAY_2:
-        return "Mon";
     case ABDAY_3:
-        return "Tue";
     case ABDAY_4:
-        return "Wed";
     case ABDAY_5:
-        return "Thu";
     case ABDAY_6:
-        return "Fri";
     case ABDAY_7:
-        return "Sat";
+        return reinterpret_cast<char const*>(short_day_names[item - ABDAY_1].bytes().data());
```

Comment:
doesn't `StringView::characters_without_null_termination()` achieve the same without a reinterpret_cast?

Reply:
Yup. Thanks for pointing that out.